.github/workflows/lint-and-test.yaml

```diff
@@ -63,21 +111,21 @@ jobs:
       - name: Run tests and report code coverage
         run: |
           # enable nightly features so that we can also include Doctests
-          RUSTC_BOOTSTRAP=1 cargo tarpaulin -o xml -o lcov -o html --doc --tests
+          RUSTC_BOOTSTRAP=1 cargo tarpaulin -o xml -o lcov -o html --doc --tests -- --test-threads 1
```

Comment:
on my local Ubuntu 22.04 machine I need to explicitly set LD_LIBRARY_PATH to also include `${VSOMEIP_INSTALL_PATH}/lib` for the tests to run successfully ...

Reply:
Added this